inc/define.php

```diff
@@ -476,6 +476,9 @@
 
 $CFG_GLPI['kanban_types']        = ['Project'];
 
+$CFG_GLPI['domain_types']        = ['Computer', 'Monitor', 'NetworkEquipment', 'Peripheral',
```

Comment:
You should add a clean logic for items in `$CFG_GLPI['domain_types']` in `CommonDBTM::cleanRelationTable()`.

Reply:
See last commit